tests/unit/test_passwords.py

```diff
@@ -0,0 +1,16 @@
+from server.application.auth.passwords import generate_api_token
+from server.infrastructure.auth.passwords import Argon2PasswordEncoder
+
+
+def test_generate_api_token() -> None:
+    api_token = generate_api_token()
+    assert len(api_token) == 64
+    assert api_token.isalnum()
+
+
+def test_argon2_password_encoder() -> None:
+    encoder = Argon2PasswordEncoder()
+    hsh = encoder.hash("s3kr3t")
```

Comment:
maybe use the variable name `hash_` to be more explicit?

Reply:
Avec plaisir :-)